spec/classes/config/audit_profiles_spec.rb

```diff
@@ -7,7 +7,16 @@
 describe 'auditd' do
   on_supported_os.each do |os, os_facts|
     context "on #{os}" do
-      let(:facts) { os_facts }
+      let(:facts){
+        if ! os_facts[:auditd_major_version]
```

Comment:
I would dup os_facts and then modify the duplicate instead of modifying os_facts